### DISCUSSION TITLE
Is there a way to log train and valid metrics separately?

### DISCUSSION THREAD

**Discussion by saravanabalagi** [Lightning-AI/pytorch-lightning#8860] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Currently I get one file `metrics.csv`

![image](https://user-images.githubusercontent.com/8567893/129092562-13c5fe36-290f-4e3d-97a5-c9f48e780df6.png)

I see that I can later split that using pandas for example

![image](https://user-images.githubusercontent.com/8567893/129094093-0059ec8a-84fb-4a50-913b-ce163e171a54.png)

But I tend to think that `metrics_{split}.csv` would be better as things are neatly organized. Is this possible?

**Comment by justusschock** [comment]
The only thing I am aware of is that you subclass  the CSV logger to implement that yourself and filter the keys. After you did that, you can use two loggers (one for each split) with our logger collection.
